<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="topBannerLabel">
     <property name="text">
      <string>Spotify login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>&amp;Username</string>
     </property>
     <property name="buddy">
      <cstring>usernameComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="usernameComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>&amp;Password</string>
     </property>
     <property name="buddy">
      <cstring>passwordLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="dmCheckBox">
     <property name="text">
      <string>I am dungeon &amp;master</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
